cdc/sink/mq/mq.go

```diff
@@ -342,6 +338,7 @@ func (k *mqSink) EmitDDLEvent(ctx context.Context, ddl *model.DDLEvent) error {
 
 // Close the producer asynchronously, does not care closed successfully or not.
 func (k *mqSink) Close(ctx context.Context) error {
+	k.flushWorker.closeMsgChan()
 	go k.mqProducer.Close()
```

Comment:
Do we need to close the `mqProducer` asynchronous?

Reply:
Required because sarama gets stuck when kafka is not healthy. So we need to process it asynchronously, otherwise it will get stuck in ticdc.

Reply:
It's better to add some comments here to make the code clear.

Reply:
I will add it.

Reply:
Thanks!

Reply:
Done